### ISSUE TITLE
Suggestion: OperationNameGenerator should be able to consider Controllers/WARs as well

### ISSUE THREAD

**Issue by mvdobrinin** [springfox/springfox#2442]
Namely because otherwise commonly used names like `list` come up as duplicates even though they are unique in their own Controller/WAR.

**Comment by dilipkrish** [comment]
It will add a number at the end to distinguish the names right?

**Comment by mvdobrinin** [comment]
@dilipkrish Yes, exactly. Any reason not to have this as a plugin similar to ParameterNameReader?
